snuba/datasets/errors_replacer.py

```diff
@@ -188,13 +205,14 @@ def set_project_needs_final(
 
 def get_projects_query_flags(
```

Comment:
We do not set the replacement type in redis when the entire project needs final `set_project_needs_final`. Is that intentional? I think we should keep track of the type of replacement even when we set the entire project to final instead of some specific groups.